frontend/src/pages/Graphing/GraphingEditor.tsx

```diff
@@ -606,6 +615,8 @@ export const GraphingEditor: React.FC = () => {
 		bucketBySetting,
 		fetchedLimitMetric,
 		expressions,
+		sqlEnabled,
+		sql,
```

Comment:
different x axis behavior

https://www.loom.com/share/3c099c860ac14787b9820a920f4a2309

Reply:
the data returned is expected in this case - interpolating / extrapolating datapoints is possible for simple queries but tricky in general, so the response will not include any buckets for which there is no data. I'm handling this on the UI side instead of returning "empty" buckets in the response - I already changed the x axis to be numeric to handle gaps in data, will change the domain start/end to be the timestamps